libbeat/docs/shared-cloudfoundry.asciidoc

```diff
@@ -0,0 +1,128 @@
+[[running-on-cloudfoundry]]
+=== Run {beatname_uc} on Cloud Foundry
+
+ifeval::["{beatname_lc}"=="filebeat"]
+You can use {beatname_uc} on Cloud Foundry to retrieve and ship logs.
+endif::[]
+ifeval::["{beatname_lc}"=="metricbeat"]
+You can use {beatname_uc} on Cloud Foundry to retrieve and ship metrics.
+endif::[]
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {beatname_uc} has not yet been
+released, no build is currently available for this version.
+
+endif::[]
+
+
+[float]
+==== Cloud Foundry credentials
+
+{beatname_uc} needs credentials created with UAA so it can connect to loggregator to receive the logs. The `uaac`
+command will create the required credentials for connecting to loggregator.
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+uaac client add {beatname_lc} --name {beatname_lc} --secret changeme --authorized_grant_types client_credentials,refresh_token --authorities doppler.firehose,cloud_controller.admin_read_only
+------------------------------------------------
+
+[WARNING]
+=======================================
+*Use a unique secret:* The `uaac` command shown here is an example. Remember to
+replace `changeme` with your secret, and update the +{beatname_lc}.yml+ file to
+use your chosen secret.
+=======================================
+
+
+[float]
+==== Cloud Foundry deploy manifests
+
+You deploy {beatname_uc} as an application with no route.
+
+Cloud Foundry requires that 3 files exist inside of a directory to allow {beatname_uc} to be pushed. The commands
+below provide the basic steps for getting it up and running.
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-linux-x86_64.tar.gz
+tar xzvf {beatname_lc}-{version}-linux-x86_64.tar.gz
+cd {beatname_lc}-{version}-linux-x86_64
+curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/cloudfoundry/{beatname_lc}/{beatname_lc}.yml
+# Update api_address, client_id, client_secret in downloaded {beatname_lc}.
+curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/cloudfoundry/{beatname_lc}/manifest.yml
+------------------------------------------------
+
+
+[float]
```

Comment:
I would remove this float attribute, so the section appears in the table of contents on the right side of the page.